### PR TITLE
Added GO_REFs regex

### PR DESCRIPTION
**Pull request by mcourtot**
Added regex for GO_REFs as per https://github.com/geneontology/go-site/issues/214

### CODE REVIEW

**Comment by mcourtot** [comment]
hm - there is an extra /go/ in the test which causes the fail. let me try and see if I can commit over my PR...

File: config/go.yml [hunk 1]
```diff
@@ -40,4 +40,14 @@ entries:
 
 - prefix: /
   replacement: http://geneontology.org/ontology/
+  
+# Terms for GO REFs
+# https://github.com/geneontology/go-site/blob/master/metadata/gorefs/goref-0000001.md
+# should point to
+# http://purl.obolibrary.org/obo/go/references/0000001
+- regex: ^/references/(\d+)$
```
Comment (jamesaoverton) [inline config/go.yml]: Confusingly unlike the others entry types, `regex` has to match the full path. It should be: `^/obo/go/references/(\d+)$`

File: config/go.yml [hunk 2]
```diff
@@ -40,4 +40,14 @@ entries:
...
+  replacement: https://github.com/geneontology/go-site/blob/master/metadata/gorefs/goref-$1.md
+  tests:
+  - from: /references/0000001
```
Comment (jamesaoverton) [inline config/go.yml]: This test should be right now -- that's the confusing part. Sorry.

**Comment by mcourtot** [comment]
Hm - still failing tests. Shall I just merge and hope for the best :) Maybe naive question, but how can the test succeed before the PURL is setup? Is it expected to fail at this stage?

**Comment by jamesaoverton** [comment]
Now the problem is that that regex entry comes after the `prefix: /` entry. The rules are matched in order, and `prefix: /` will match everything starting with `http://purl.obolibrary.org/obo/go/`.

Move the regex somewhere before `prefix: /` and it should be good.

**Comment by mcourtot** [comment]
Yes - all passed (finally!). Thanks again for the help. Can you merge if it all looks ok to you as well?

**Comment by jamesaoverton** [comment]
Sorry for the hassle. I admire your perseverance! :^)

**Pull request merged**